R/boxes.R

```diff
@@ -250,7 +251,7 @@ infoBox <- function(title, value = NULL, subtitle = NULL,
 #' @export
 box <- function(..., title = NULL, footer = NULL, status = NULL,
                 solidHeader = FALSE, background = NULL, width = 6,
-                height = NULL, collapsible = FALSE, collapsed = FALSE) {
+                height = NULL, collapsible = FALSE, collapsed = FALSE, wrench = FALSE) {
 
   boxClass <- "box"
```

Comment:
After deep thinking: 
Not so much sure if `wrench=` should be boolean. Rather something like `tags$li = `....